docu/files/BuilderInputs.html.md

```diff
@@ -3,7 +3,7 @@ title: 'Client-side field validation in the Builder'
 service: 'Builder SDK'
 area: 'Core'
 type: Tutorial
-order: 7
+order: 8
 ---
 
 The **builder.inputs** Angular module provides a mechanism for client-side field validation. The **builder.inputs** module contains the <b>builder-input</b> Angular directive, which enables you to validate any input or text area field.
```

Comment:
Per the guidelines, use "enable" when referring to the application, and "allow" when referring to something a user can do: "the <b>builder-input</b> Angular directive, which allows you to validate any input or text area field."